my-first-project/src/components/Card.js

```diff
@@ -4,82 +4,94 @@ import { BsPencilSquare } from "react-icons/bs";
 import { BsCheckLg } from "react-icons/bs";
 import { FcCancel } from "react-icons/fc";
 
-function Card() {
-  const [checked, setChecked] = useState();
-  const checkboxHandler = () => {
-    setChecked(!checked);
+function Card(props) {
+  const cardCheckboxHandler = () => {
+    setCardChecked(!cardChecked);
   };
-  const defaultTitle = "Info";
-  const defaultText =
-    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, \
-  sed do eiusmodtempor incididunt ut labore et dolore magna aliqua.";
+  const defaultTitle = props.title;
+  const defaultText = props.text;
+  const viewOnly = props.viewOnly;
+  const setEditMode = props.setEditMode;
+  const editMode = props.editMode;
```

Comment:
Can you please refactor code this way: destructurize props on 7-th line, pass only one method for card updating in props, move 'checked' field of card to cards array.